packages/terra-pills/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Pills
+
+[![NPM version](https://badgen.net/npm/v/terra-pills)](https://www.npmjs.com/package/terra-pills)
+[![Build Status](https://badgen.net/travis/cerner/terra-framework)](https://travis-ci.com/cerner/terra-framework)
+
+The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.
```

Comment:
I think this description should be re-worked: this is the first time I've seen the readme and the concept description didn't help. I will offer some suggestions once I figure out what pills is - first I need to read more of the PR to know that.

Reply:
changed ae07eb46c1332427df41f2b68be6f9007c75930c